products/cfengine.md

```diff
@@ -0,0 +1,46 @@
+---
+title: CFEngine
+layout: post
+command: cf-agent --version
+chagnelogTemplate: https://github.com/cfengine/core/blob/__RELEASE_CYCLE__.x/ChangeLog
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+activeSupportColumn: true
+eolColumn: Supported
+iconSlug: NA
+permalink: /cfengine
+
```

Comment:
^^ it will be nice to add link: tag to products webpage

Reply:
link is not mandatory if you dont want to add it i will still give OK to this PR

Reply:
ack

Reply:
thx for link :)